### PR TITLE
fix(DB/Command): fix help typo 'trigered' for .cast commands

### PR DESCRIPTION
**Pull request by sogladev**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

data from `command` `name like 'cast%'`
```
name  help
cast	"Syntax: .cast #spellid [triggered]
  Cast #spellid to selected target. If no target selected cast to self. If 'trigered' or part provided then spell casted with triggered flag."
cast back	"Syntax: .cast back #spellid [triggered]
  Selected target will cast #spellid to your character. If 'trigered' or part provided then spell casted with triggered flag."
cast dest	"Syntax: .cast dest #spellid #x #y #z [triggered]
  Selected target will cast #spellid at provided destination. If 'trigered' or part provided then spell casted with triggered flag."
cast dist	"Syntax: .cast dist #spellid [#dist [triggered]]
  You will cast spell to pint at distance #dist. If 'trigered' or part provided then spell casted with triggered flag. Not all spells can be casted as area spells."
cast self	"Syntax: .cast self #spellid [triggered]
Cast #spellid by target at target itself. If 'trigered' or part provided then spell casted with triggered flag."
cast target	"Syntax: .cast target #spellid [triggered]
  Selected target will cast #spellid to his victim. If 'trigered' or part provided then spell casted with triggered flag."
```

https://github.com/azerothcore/azerothcore-wotlk/blob/85965feab65a03b5479606429b6a52286d03d014/src/server/scripts/Commands/cs_cast.cpp#L89

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1718702527397206595.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `command` SET `help` = REPLACE(`help`, 'trigered', 'triggered') WHERE `name` like 'cast%';
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1718702527397206595.sql:2]: I would prefer you to update the entire text instead of using Replace.

Comment (sogladev) [reply data/sql/updates/pending_db_world/rev_1718702527397206595.sql]: caught another typo doing it with inserts :laughing:

**Review by sudlud — APPROVED** [review]

**Comment by Nyeriah** [comment]
While you are at, mind to fix some broken English? "Cast" is a irregular verb therefore the proper past tense is *cast* and not *casted*
